src/displayBackend/drm/Dumb.cpp

```diff
@@ -150,16 +150,16 @@ void DumbDrm::copy()
 	}
 
 	DLOG(mLog, DEBUG) << "Copy dumb, handle: " << mBufDrmHandle;
-	if(mStride == mFrontStride)
+	if (mGnttabBuffer->size() == mSize)
 	{
 		memcpy(mBuffer, mGnttabBuffer->get(), mSize);
 		return;
 	}
-	auto src = reinterpret_cast<unsigned char*>(mGnttabBuffer->get());
-	auto dst = reinterpret_cast<unsigned char*>(mBuffer);
-	for(unsigned int i = 0; i < mHeight; i++)
+	auto src = reinterpret_cast<uint8_t*>(mGnttabBuffer->get());
```

Comment:
use static_cast<uint8_t*>(mGnttabBuffer->get()); and static_cast<uint8_t*>(mBuffer);